src/__test__/Details.test.js

```diff
@@ -0,0 +1,50 @@
+import { BrowserRouter, useParams } from 'react-router-dom';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import Details from '../components/Details';
+import '@testing-library/jest-dom';
+
+const mockStore = configureMockStore({});
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useParams: jest.fn(),
+}));
+let store;
+beforeEach(() => {
+  store = mockStore({
+    devices: {
+      devices: [
+        {
+          name: 'Light Based Imaging',
+          class: '2',
+          code: 'PSN',
+          speciality_area: 'SU',
+          submission_type: '1',
+          speciality_description: 'General, Plastic Surgery',
+          id: '1',
+        },
+      ],
+      isLoading: false,
+      error: null,
+    },
+  });
+  store.dispatch = jest.fn();
+  useParams.mockReturnValue({
+    id: '1',
+  });
```

Comment:
[OPTIONAL] All the tests are passing, but I encourage you to make sure that the mocked data are still to an extent match with the one currently used in the application for consistency purposes.

Reply:
Ok, I will correct it. Thank you!!